code/game/objects/items/storage/smartpack.dm

```diff
@@ -201,19 +216,26 @@
 	to_chat(user, SPAN_INFO("The current charge reads [battery_charge]/[SMARTPACK_MAX_POWER_STORED]"))
 	update_icon(user)
 
+	var/filter_size = EXOSKELETON_OFF_FILTER_SIZE
+	var/filter_color = PROTECTIVE_FORM_COLOR
+	if(show_exoskeleton)
+		filter_size = EXOSKELETON_ON_FILTER_SIZE
+	user.add_filter("synth_protective_form", 1, list("type" = "outline", "color" = filter_color, "size" = filter_size))
```

Comment:
named args for 1 and the list

Reply:
bruh thats nowhere else in code!!!!!!!!!!!! how inconsistent!!!!!!!!!!!!!!

Reply:
but it's in standards and styles